### PR TITLE
FIxed issue No. 747

### PR DESCRIPTION
**Pull request by onkar-shaligram**
No proper Horizontal Letter Spacing #747
Increased the Horizontal Letter Spacing

Before:--
Screenshot (41)_LI

After:--
Screenshot (42)_LI

### CODE REVIEW

**Comment by onkar-shaligram** [comment]
@Dishebh @liveHarshit please have a look at this preview link https://fervent-wiles-6ec432.netlify.app/   To confirm my changes

**Comment by Dishebh** [comment]
Please resolve merge conflicts.

**Comment by onkar-shaligram** [comment]
@Dishebh Merge conflicts resolved. Please verify

**Comment by Dishebh** [comment]
Great. Try writing commit messages that tell what changes you have done.

**Review by Dishebh — APPROVED** [review]
LGTM

**Comment by onkar-shaligram** [comment]
@Dishebh Sorry sir for not writing commit messages. But will write commit messages henceforth..

File: css/theme-lava.css [hunk 1]
```diff
@@ -765,7 +766,11 @@ nav .container {
 }
 .copy-text-bottom {
   left: 24px;
+
+  bottom:25%;
+
   bottom:40px;
+
```
Comment (liveHarshit) [inline css/theme-lava.css]: Remove all extra lines

**Review by liveHarshit — CHANGES_REQUESTED** [review]

File: css/theme-lava.css (hunk 1 above)
Comment (onkar-shaligram) [reply css/theme-lava.css]: Done Sir.

**Comment by onkar-shaligram** [comment]
Updated changes as requested by @liveHarshit . Please review.

**Review by liveHarshit — APPROVED** [review]

**Comment by onkar-shaligram** [comment]
Thankyou @Dishebh @liveHarshit for the merge to be Successful. It had been a great experience working on this PR. 👍😀

**Pull request merged**